tests/clickhouse-test

```diff
@@ -1447,8 +1452,7 @@ class TestCase:
         description_full = messages[result.status]
         description_full += self.print_test_time(result.total_time)
         if result.reason is not None:
-            description_full += " - "
-            description_full += result.reason.value
+            description_full += f"\nReason: {result.reason.value} "
```

Comment:
Why you prefer to use `\n` it became too verbose, let's get compact behavior back - https://github.com/ClickHouse/ClickHouse/pull/75530